Gemfile

```diff
@@ -26,7 +26,7 @@ group :development, :test do
   gem 'rake'
   gem 'rdoc'
   gem 'rspec', '~> 3.0'
-  gem 'rubocop', '~> 0.59', require: false
+  gem 'rubocop', '0.60', require: false
```

Comment:
please don't lock it, minor rubocop updates are much easier to handle then big ones

Reply:
@dblock Could you guys agree on a decision? I'm happy either way as I see both views as valid, but I don't want to step over either of you

Reply:
Lets take it to #719